visualizations/load-aware/plotting/comparison.py

```diff
@@ -86,9 +86,13 @@ def ExecutionTimeComparison_generatePodTimes(entry, variables, exclude_workload=
 
 def ExecutionTimeComparison_generateStats(entry, variables, data):
     df = pd.DataFrame(data)
-    q1, med, q3 = stats.quantiles(df.Duration)
-    q90 = stats.quantiles(df.Duration, n=10)[8] # 90th percentile
-    q100 = df.Duration.max()
+    q1, med, q3 = (0.0, 0.0, 0.0)
+    q90 = 0.0
+    q100 = 0.0
+    if len(df) >= 2:
```

Comment:
I didn't look into the data model, so I'm not sure why we need len(df) >=2 but I'm assuming you know what you're doing.  Similarly below.

Reply:
Having less than 2 data points causes the stats functions to panic, so we avoid calling them if this is the case.

[Some previous discussion](https://github.com/openshift-psap/ci-artifacts/pull/863#issuecomment-1653385656)